### ISSUE TITLE
Editor: Pasting in Chinese text causes editor to crash (simple)

### ISSUE THREAD

**Issue by CompChris** [Automattic/wp-calypso#56144]
### Steps to reproduce the behavior

Chat: 30552338-hc
Followup: 4285868-zen

1. With the locale zn-cn
2. Pasting into the editor Chinese text causes the editor to crash. User is pasting from Word, if that matters.

![image](https://user-images.githubusercontent.com/15944096/133214108-3a20dfd0-fdcd-4418-b85c-d68c6f5526bb.png)

#### What I expected to happen

It should not crash when this text is pasted in.

#### What actually happened

Guessing this is related to Gutenberg update. (simple site)

Customer used the following Browsers/versions:

- Google Chrome Version 93.0.4577.63（Official Version） （64 bit）
- Microsoft Edge Version 93.0.961.44 （Official Version） （64 bit）

#### Does this happen on simple or atomic sites or both?

Simple. But wouldn't be surprised if this happens on Atomic too.

#### Is there any console output or error text?

```
TypeError: Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'.
    at pe (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/build/rich-text/index.min.js?m=1630518223h&ver=8f70a366b9cae91a6f5753db0674024a:1:18233)
    at fe (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/build/rich-text/index.min.js?m=1630518223h&ver=8f70a366b9cae91a6f5753db0674024a:1:17172)
    at Ee (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/build/rich-text/index.min.js?m=1630518223h&ver=8f70a366b9cae91a6f5753db0674024a:1:18650)
    at xe (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/build/rich-text/index.min.js?m=1630518223h&ver=8f70a366b9cae91a6f5753db0674024a:1:19081)
    at b (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/build/rich-text/index.min.js?m=1630518223h&ver=8f70a366b9cae91a6f5753db0674024a:1:30088)
    at C (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/build/rich-text/index.min.js?m=1630518223h&ver=8f70a366b9cae91a6f5753db0674024a:1:30848)
    at https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/build/rich-text/index.min.js?m=1630518223h&ver=8f70a366b9cae91a6f5753db0674024a:1:31049
    at Bj (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/vendor/react-dom.min.edd8b7a6.js?m=1624062436h:7:29846)
    at Qj (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/vendor/react-dom.min.edd8b7a6.js?m=1624062436h:9:876)
    at unstable_runWithPriority (https://s0.wp.com/wp-content/plugins/gutenberg-core/v11.4.1/vendor/react.min.755b0ae9.js?m=1624062436h:5:3609)
```

#### Level of impact (Does it block purchases? Does it affect more than just one site?)

It likely affects all sites in Chinese.

**Comment by DavidSaladu** [comment]
Updated with Browsers information.

**Comment by Greatdane** [comment]
I have tried to reproduce this (as have multiple people) without success. 

Updated the ticket with the documents providerd by user in case anyone else wants to try; 4285868-zd-woothemes

**Comment by formosattic** [comment]
Updated initial post with screenshot.

Additional notes:
* user is copying from Word (see example in 4285868-zd-woothemes)
* also unable to reproduce
* user will try copying from Word to another editor (Google Doc, Notepad) to WordPress.com

Set `Needs Author Reply label`, updated based on ticket reply.

**Comment by Robertght** [comment]
I can confirm I wasn't able to replicate this either.

 - I also switched the language of the site to Chinese, but no relevant results.
 - I tried it from a Windows machine with the Office app - that opened the web office app, but no errors there either.

**Comment by Greatdane** [comment]
The user has posted two videos on the ticket, showing the error happening. The first video they copy from a Word Document, but it also looks like they are copying the ending 'Enter' sign. So that could be related to the [obj] bug?
<img width="135" alt="Markup 2021-09-17 at 15 45 21" src="https://user-images.githubusercontent.com/5075840/133737065-dbf32348-d1f8-41ea-8843-09ab690ac3d7.png">

Videos available in 4285868-zd-woothemes

**Comment by Robertght** [comment]
>The user has posted two videos on the ticket, showing the error happening. The first video they copy from a Word Document, but it also looks like they are copying the ending 'Enter' sign. So that could be related to the [obj] bug?

I couldn't see that in the web version of the doc. I believe it could be only a app-related feature of sort.

@getdave Could this be something that is be caused by the same issue https://github.com/WordPress/gutenberg/issues/34602#issuecomment-915308888 ?

**Comment by getdave** [comment]
> @getdave Could this be something that is be caused by the same issue WordPress/gutenberg#34602 (comment) ?

I'm not sure. I know @gwwar and @ellatrix we're looking at WordPress/gutenberg#34602 so they may (almost certainly!) be able to speak with greater authority.

**Comment by gwwar** [comment]
If folks are copying from word it's likely also adding the object replacement character. That fix should be available in the 11.6 release. If folks find other combinations of text that break the editor, please add the example content to this issue.

**Comment by dpasque** [comment]
It seems like this remains unreproducible, and the potentially related issue has been fixed in recent Gutenberg upgrades, so closing this issue for now.

If this crops up again, please feel free to re-open!
